composer.json

```diff
@@ -16,11 +16,11 @@
     ],
     "require": {
         "php": "^7.1",
```

Comment:
What about requiring `php: ~7.2`?
7.1 EOL is in "1st Dec 2019"

Reply:
yeah makes sense :+1: